src/NodaTime.Web/Startup.cs

```diff
@@ -80,6 +84,43 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
                 ContentTypeProvider = new FileExtensionContentTypeProvider
                 {
                     Mappings = { [".nzd"] = "application/octet-stream" }
+                },
+                OnPrepareResponse = (context) => {
+                    // Set the Cache-Control header for static content.
+                    var headers = new ResponseHeaders(context.Context.Response.Headers);
+
+                    // All static content is cacheable by intermediates.
+                    headers.CacheControl = new CacheControlHeaderValue()
```

Comment:
I'm wondering whether I should just remove this, as I'm not sure that "Cache-Control: public" with no max-age is that useful. (I added this originally when I thought I could create the object once; see below.)